examples/machine-learning/a3-highgpu-8g/README.md

```diff
@@ -181,6 +181,15 @@ blueprint matches the name of the user-created reservation.
   a3_maintenance_interval: ""
 ```
 
+### Using Spot VM or DWS Flex
+
+> [!IMPORTANT]
+> Only set one of the property : either spot vm , dws flex or reservation
```

Comment:
switch "set one of the property" to "select one of the provisioning models" and show how to do "input reservation name or set spot/flex to true" for example